Library/Homebrew/cmd/vendor-install.sh

```diff
@@ -39,10 +39,10 @@ fetch() {
 
   if [[ -n "$HOMEBREW_QUIET" ]]
   then
-    curl_args+=(--silent)
+    curl_args[${#curl_args[*]}]="--silent"
```

Comment:
This syntax is awful (Bash's fault not yours). Is it worth just avoiding Bash arrays here in favour of space separated strings?